code/game/objects/items/devices/instruments.dm

```diff
@@ -268,3 +268,38 @@
 	item_state = "electronic"
 	lefthand_file = 'icons/mob/inhands/misc/devices_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/misc/devices_righthand.dmi'
+
+/obj/item/musicbeacon
+	name = "express delivery beacon"
+	desc = "Summon your tool of art."
+	icon = 'icons/obj/device.dmi'
+	icon_state = "gangtool-red"
+	item_state = "radio"
+	var/static/list/instruments

```

Comment:
You only use this once and only while making `display_names`. This should not be defined on the type and just be a static var inside the if block making `display_names`